Cython/Plex/Errors.py

```diff
@@ -25,7 +23,9 @@ class InvalidRegex(PlexError):
 
 class InvalidToken(PlexError):
     def __init__(self, token_number, message):
-        PlexError.__init__(self, "Token number %d: %s" % (token_number, message))
+        msg = ("Token number {number}: {message}"
+               .format(number=token_number, message=message))
```

Comment:
Aaah! Please don't switch to `.format()` formatting!
We use '%' formatting throughout the code base – for various (sometimes historical) reasons, but above all for better looking format strings of C's "{…}" blocks.